src/pages/instances/forms/ConvertToInstanceForm.tsx

```diff
@@ -0,0 +1,181 @@
+      );
+      const instanceName = truncateInstanceName(sanitisedFileName, suffix);
+      await formik.setFieldValue("imageFile", file);
+      await formik.setFieldValue("name", instanceName);
+
+      // validate instance name
+      await formik.validateField("name");
+      void formik.setFieldTouched("name", true, true);
+      if (!formik.errors.name) {
+        formik.setFieldError("name", undefined);
+      }
+    }
+  };
+
+  const handleCloseModal = useCallback(() => {
+    formik.resetForm();
+    close();
+  }, [formik.resetForm, close]);
+
+  return (
+    <Modal
+      close={handleCloseModal}
+      className="convert-to-instance-modal"
+      title={
+        uploadState ? "Uploading image to LXD" : "Upload virtual machine image"
+      }
+      buttonRow={
+        <>
+          <Button
+            appearance="base"
+            className="u-no-margin--bottom"
+            type="button"
+            onClick={handleCloseModal}
+          >
+            Cancel
+          </Button>
+          <ActionButton
+            appearance="positive"
+            className="u-no-margin--bottom"
+            loading={isLoading}
+            disabled={disableSubmit}
+            onClick={() => void formik.submitForm()}
+          >
+            Upload and create
+          </ActionButton>
+        </>
+      }
+    >
+      <Form
+        onSubmit={formik.handleSubmit}
+        className={uploadState ? "u-hide" : ""}
+      >
+        <Input
+          id="image-file"
+          name="imageFile"
+          type="file"
+          label="Image file"
+          onChange={(e) => void handleFileChange(e)}
+        />
+        <Input
+          {...formik.getFieldProps("name")}
+          id="name"
+          type="text"
+          label="New instance name"
+          placeholder="Enter name"
+          error={formik.touched.name ? formik.errors.name : null}
+          disabled={!formik.values.imageFile}
+        />
+        <StoragePoolSelector
+          project={project}
+          value={formik.values.pool}
+          setValue={(value) => void formik.setFieldValue("pool", value)}
+          selectProps={{
+            id: "pool",
+            label: "Root storage pool",
+            disabled: !project || !formik.values.imageFile,
+          }}
+        />
+        <Select
+          {...formik.getFieldProps("architecture")}
+          id="architecture"
+          label="Image architecture"
+          options={supportedVMArchOptions}
+          disabled={!formik.values.imageFile}
+        />
+        <label htmlFor="">Conversion options</label>
+        <div className="conversion-options">
+          <Input
+            {...formik.getFieldProps("formatConversion")}
+            type="checkbox"
+            label={
+              <>
+                Format{" "}
+                <Icon
+                  name="information"
+                  title="Check this option to convert the image into raw format"
```

Comment:
```suggestion
                  title="Convert the image into raw format. This can be skipped if the image is already in raw format to speed up the import."
```
Not sure if the 2nd part is actually true. I'm assuming it is?

Reply:
Yes that's correct, the backend will simply skip the conversion step for raw images

Reply:
Assuming the above is true, then the label of the checkbox should read "Convert the image into raw format" and the tooltip contain the 2nd sentence.

Reply:
Okay I'm going to vertically stack the checkboxes since the label is quite long :+1: